packages/at_secondary_server/test/update_verb_test.dart

```diff
@@ -703,16 +703,45 @@ void main() {
           localLookUpResponse, localLookVerbParam, atConnection);
       expect(localLookUpResponse.data, 'hyderabad');
     });
+
+    test('test auto_notify notification expiry', () async {
+      SecondaryKeyStore keyStore = keyStoreManager.getKeyStore();
+      UpdateVerbHandler updateHandler = UpdateVerbHandler(keyStore);
+      UpdateVerbHandler.setAutoNotify(true);
+      AtMetaData metaData = AtMetaData()..ttl = 1000;
+      AtNotification? autoNotification;
+
+        autoNotification = await updateHandler.notify('@from', '@to', 'na',
+            'na-value', NotificationPriority.high, metaData);
+      int ttlInMillis =
+          Duration(minutes: AtSecondaryConfig.notificationExpiryInMins)
+              .inMilliseconds;
+      DateTime notifExpiresAt = autoNotification!.notificationDateTime!
+          .toUtc()
+          .add(Duration(milliseconds: ttlInMillis));
+
+      expect(autoNotification.id, isNotNull);
+      expect(autoNotification.ttl, ttlInMillis);
+      //autoNotification.expiresAt and notifExpiresAt have the difference of a
+      // couple of milli seconds and they cannot asserted to be equal
+      expect(autoNotification.expiresAt?.year, notifExpiresAt.year);
+      expect(autoNotification.expiresAt?.month, notifExpiresAt.month);
+      expect(autoNotification.expiresAt?.day, notifExpiresAt.day);
+      expect(autoNotification.expiresAt?.hour, notifExpiresAt.hour);
+      expect(autoNotification.expiresAt?.minute, notifExpiresAt.minute);
+      expect(autoNotification.expiresAt?.second, notifExpiresAt.second);
```

Comment:
Same request as above

Reply:
this change has been made as part of [7dc827f](https://github.com/atsign-foundation/at_server/pull/1241/commits/7dc827f5a9ef7b2c738c0ccf78c45a29c114a368)